managed/models/postgresql_version_test.go

```diff
@@ -78,7 +78,7 @@ func TestGetPostgreSQLVersion(t *testing.T) {
 	for _, tc := range testCases {
 		tc := tc
 		t.Run(tc.name, func(t *testing.T) {
-			t.Parallel()
+			// t.Parallel()
```

Comment:
Debug leftover or we have problem with these tests?

Reply:
we have problems with these tests

Reply:
fixed